React/Modules/RCTUIManager.m

```diff
@@ -1519,12 +1519,18 @@ static void RCTMeasureLayout(RCTShadowView *view,
   RCTAssertMainQueue();
 
   // Don't use RCTScreenSize since it the interface orientation doesn't apply to it
-  CGRect screenSize = [[UIScreen mainScreen] bounds];
+  id mainScreen = [UIScreen mainScreen];
```

Comment:
There's a bug on iPad with split screen, Dimensions.get('window') always returns screen size.

window's: width & height should be UIWindow's bounds as apple said:

> Likewise, in iOS 9 it’s no longer appropriate to employ the screen’s bounds to gauge your app’s visible area. Instead, obtain the value of your app’s window’s bounds property (which the UIWindow class inherits from the UIView class.)

[https://developer.apple.com/library/ios/documentation/WindowsViews/Conceptual/AdoptingMultitaskingOniPad/index.html#//apple_ref/doc/uid/TP40015145-CH3-SW1]()

and what about provide both window and screen like android do?